docs/modules/deployment/pages/core/centos/initialize-core.adoc

```diff
@@ -0,0 +1,50 @@
+.Configure PostgreSQL database access
+
+You can store your PostgreSQL credentials in the secure credentials vault.
+To do this, use the `scvcli` command line utility to add the credentials, and reference these credentials in `opennms-datasources.xml`:
+
+. Create encrypted credentials for the PostgreSQL database connections:
++
+[source, console]
+----
+sudo -u opennms /opt/opennms/bin/scvcli set postgres opennms password <1>
+history -d $((HISTCMD-1)) && history -w <2>
+sudo -u opennms /opt/opennms/bin/scvcli set postgres-admin postgres password <1>
+history -d $((HISTCMD-1)) && history -w <2>
+----
+<1> Replace both `password` instances with the actual passwords for your `opennms` and `postgres` database accounts.
+<2> Remove previous command with sensitive information from bash history
```

Comment:
I prefer to keep the instructions simple.  I'd wager the majority of users don't care if the password is in the shell history, even if they _should_.  See also:  https://opennms.atlassian.net/browse/NMS-19033

```suggestion
sudo -u opennms /opt/opennms/bin/scvcli set postgres opennms password <1>
sudo -u opennms /opt/opennms/bin/scvcli set postgres-admin postgres password <1>
----
<1> Replace both `password` instances with the actual passwords for your `opennms` and `postgres` database accounts.
NOTE: The password may be visible in the shell history and should be removed if this is a concern.
```

Reply:
gone